src/hyperfine/benchmark.rs

```diff
@@ -234,13 +234,14 @@ pub fn run_benchmark(
     );
 
     // Run init command
-    let prepare_cmd = options
-        .preparation_command
-        .as_ref()
-        .map(|preparation_command| match cmd.get_parameter() {
-            Some((param, value)) => Command::new_parametrized(preparation_command, param, value),
-            None => Command::new(preparation_command),
-        });
+    let mut prepare_cmd = None;
```

Comment:
instead of using `mut` here, I would prefer
``` rust
let prepare_cmd = if … {
  …
} else {
  None
};
```
or (even better) a `.map` over the `options.preparation_command`, like it was before.